src/constans/VALIDATIONS.ts

```diff
@@ -0,0 +1,31 @@
+export const EmailValidation = {
+  required: "Email is required",
+  pattern: {
+    value: /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/,
+    message: "Please enter a valid email",
+  },
+};
+
+const RegExp = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[\W_]).{4,}$/;
```

Comment:
We can also add another regex for the phone number.
This website contain some usefull regular expressions  and it has multiple ones for phones:
https://uibakery.io/regex-library/phone-number